web/src/components/RemoteContentSandBox.ts

```diff
@@ -144,6 +144,38 @@ const RemoteContentSandBox = styled.div<{ $centered: boolean; $smallText: boolea
     font-weight: bold;
   }
 
+  .contact-card {
+    display: inline-block;
+    box-sizing: border-box;
+    padding: 1rem;
+    border-radius: 0.25rem;
+    background-repeat: no-repeat;
+    background-color: rgb(127 127 127 / 15%) !important;
+    background-image: linear-gradient(to right, rgb(255 255 255 / 90%) 0 100%),
+      url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij48cGF0aCBkPSJNMTMuMDE4IDEyLjQ4aC0yLjAzNkE5LjA5IDkuMDkgMCAwIDAgMS45MiAyMS42YS40OC40OCAwIDAgMCAuNDguNDhoMTkuMmEuNTMuNTMgMCAwIDAgLjQ4LS41MzggOS4wOCA5LjA4IDAgMCAwLTkuMDYyLTkuMDYyTTE2LjggNi43MmE0LjggNC44IDAgMCAxLTQuOCA0LjggNC44IDQuOCAwIDAgMS00LjgtNC44IDQuOCA0LjggMCAwIDEgNC44LTQuOCA0LjggNC44IDAgMCAxIDQuOCA0LjgiLz48L3N2Zz4=') !important;
+    background-blend-mode: difference;
+    background-position: calc(100% + 2em) calc(100% + 1em);
+    background-size: 7em;
+    box-shadow: 0 0.1em 0.1em rgb(0 0 0 / 40%);
```

Comment:
:question: Hmm, my css knowledge is a little limited, but it seems to me like we are mixing for different units for styling here. `rem`, `em`, `px` and `%`. Is this necessary or could we stick to one?

Reply:
I'm trying to be as close as the styling that is porvided here: https://github.com/digitalfabrik/integreat-app/issues/3010#issue-2704905267

Reply:
I will convert these to pixels then...